.github/workflows/cd.yml

```diff
@@ -41,12 +41,15 @@ jobs:
     uses: init4tech/actions/.github/workflows/solidity-deployment.yml@main
     needs: auto-release
     permissions:
-      contents: read
+      contents: write
       id-token: write
     with:
       network: holesky
       environment: dev
       forge-deployment-script: DeployZenith
+      deployed-contract: Zenith.s.sol
```

Comment:
why is this named `deployed-contract` - it looks like it's the name of the script file that's being run

Reply:
specifically it would be this path: `Transactions saved to: /home/runner/work/zenith/zenith/broadcast/Zenith.s.sol/17000/run-latest.json
` 

What would make the most sense to name this from your perspective

Reply:
changed to look like this:
```
      forge-deployment-contract: DeployZenith
      forge-deployment-script-file: Zenith.s.sol
      forge-deployment-signature: "run()"
```